2021q2/kernel-sanitizers.adoc

```diff
@@ -0,0 +1,31 @@
+=== Kernel Sanitizers
+
+Contact: Mark Johnston <markj@FreeBSD.org>
+
+Sanitizers are a bug-detection facility which use a combination of instrumentation inserted by the compiler (LLVM in this case) and runtime state tracking to detect bugs in C code.
```

Comment:
Not a native speaker, but my instinct tells me there shouldn't be a hyphen there.

Also, does "use" have to agree in number with "sanitizers", or with "facility" as it feels to me? If the latter, it should be "uses".

Reply:
I suspect that both should just be plural.

Reply:
That would work too.

Reply:
Thanks for pointing these things out. As a native speaker, my writing is mostly guided by intuition rather than any formal understanding of English grammar, but I do prefer to avoid mistakes. :)